src/_data/whats-new.yml

```diff
@@ -4,8 +4,26 @@ description:
   We exclude from this list proofreading, spelling checks, and all minor updates.
 link: /whats-new.html
 thread: /whatsnew-feed.xml
-updated: Thu May 28 15:55:01 2020
+updated: Tue Jun 9 10:53:40 2020
 entries:
+- description: Removed the Extension Manager section of the Software Update Guide
+    and redirected to [CLI installation](https://devdocs.magento.com/extensions/install/).
+  versions: 2.3.x
+  type: Major Update
+  date: June 3, 2020
+  link: https://github.com/magento/devdocs/pull/7343
+- description: Added a brief note about a new known issue with the [Extension Manager
+    showing no extensions in Magento Commerce 2.3.x](https://support.magento.com/hc/en-us/articles/360043980352).
+  versions: 2.3.x
+  type: Major Update
+  date: June 3, 2020
+  link: https://github.com/magento/devdocs/pull/7344
+- description: Published documentation deliverables for the Commerce and Open Source
```

Comment:
Link in description?

Reply:
Since there were so many updated topics, I didn't think adding a link made sense. I supposed we could point to release notes.

Reply:
I agree, no need to add a link for major site updates that affected multiple topics in different categories.

Reply:
👍